src/main/java/org/sonarlint/intellij/core/EmbeddedPlugins.kt

```diff
@@ -58,7 +62,11 @@ object EmbeddedPlugins {
         EmbeddedPlugin(Language.RUBY, "Ruby", "sonar-ruby-plugin-*.jar"),
         EmbeddedPlugin(Language.XML, "XML", "sonar-xml-plugin-*.jar"),
         EmbeddedPlugin(Language.SECRETS, "Secrets detection", "sonar-text-plugin-*.jar"),
-        EmbeddedPlugin(Language.GO, "Go", "sonar-go-plugin-*.jar")
+        EmbeddedPlugin(Language.GO, "Go", "sonar-go-plugin-*.jar"),
+        EmbeddedPlugin(Language.CLOUDFORMATION, "CloudFormation", "sonar-iac-plugin-*.jar"),
+        EmbeddedPlugin(Language.DOCKER, "Docker", "sonar-iac-plugin-*.jar"),
+        EmbeddedPlugin(Language.KUBERNETES, "Kubernetes", "sonar-iac-plugin-*.jar"),
+        EmbeddedPlugin(Language.TERRAFORM, "Terraform", "sonar-iac-plugin-*.jar")
```

Comment:
I think we should find a way to simplify that, basically we are defining 4 times the same plugin as embedded